runtime/crab/src/pallets/evm_.rs

```diff
@@ -52,16 +53,23 @@ impl<R: darwinia_evm::Config> PrecompileSet for CrabPrecompiles<R> {
 	}
 }
 
+pub struct FixedGasPrice;
+impl FeeCalculator for FixedGasPrice {
+	fn min_gas_price() -> U256 {
+		U256::from(1 * COIN)
```

Comment:
@AsceticBear @hujw77  will 1 coin be too large or small?

Reply:
`1 COIN = 1e9` it's right?

Reply:
yes，but coin here is kind of miss leading? it is 1gwei and 10^-9 RING cause in evm the decimal is 10^18? how may gas for transfer ether? 21000？

Reply:
See the screenshot here https://github.com/darwinia-network/darwinia/pull/765#issue-1037402454

Reply:
@AurevoirXavier Please change to 10 GWEI, and replace "COING" with "GWEI".

GWEI = 10^9